### ISSUE TITLE
Add horizontal navigation bar below header for very large docs (optional)

### ISSUE THREAD

**Issue by squidfunk** [squidfunk/mkdocs-material#132]
**TL;DR: a new, optional, top navigation bar for the first level items**

Some documentation projects are huge and contain sections with many entries. For these projects it could be better to add the main sections, e.g. "Home", "Developer Guide" and "API Reference" to a top header bar and show the sub entries on the left. This should only apply to anything above tablet, because it makes no sense when there is the side navigation. Here is a proof of concept:

<img width="1439" alt="bildschirmfoto 2017-01-26 um 23 00 15" src="https://cloud.githubusercontent.com/assets/932156/22352224/8aea95e4-e41b-11e6-9881-4be30de200ac.png">

For the Material docs, this makes no sense, as there are many pages on the top that have no children. It's only useful for docs that have mainly sections with many pages below them.

The new navigation bar would not be fixed. It hides when scrolling:

<img width="1439" alt="bildschirmfoto 2017-01-26 um 23 04 01" src="https://cloud.githubusercontent.com/assets/932156/22352296/cfd4a208-e41b-11e6-85ef-eca2a4681060.png">

Subject to change. Any thoughts are welcome.

**Comment by makshh** [comment]
It's just an idea to make content container wider.

![Material](http://s24.postimg.org/vq4twni0z/111.png)

**Comment by squidfunk** [comment]
Thanks for your idea. My thoughts on it:

* There will definitely be no sub-tab navigation - only one layer. I think a second layer does not really improve the feel of the template. It also cannibalizes the side navigation.
* I don't want to widen the container, because then the line length is wider than 80-90 characters, which is the optimum for reading. This is one reason I made the content container narrower from 0.2 to 1.x.

**Comment by sb4092** [comment]
Could a horizontal menu and submenu be added as an alternative to a horizontal menu and side menu? This would be very useful for large text heavy websites e.g. procedure manuals with sections, subsections and sub-subsections.

**Comment by squidfunk** [comment]
@sb4092 I won't add this second horizontal navigation bar to the Material theme, as it doesn't work out in my opinion, but the template blocks and partials are all well-defined, so it should be pretty easy to extend the theme to do that.

**Comment by sb4092** [comment]
OK, thanks for the feedback. I'll give it a go
 

Sent: Wednesday, February 01, 2017 at 2:32 AM

**Comment by brendo** [comment]
I look forward to this feature and will happily test or contribute when it's available 👍

**Comment by squidfunk** [comment]
Implemented in #187.

**Comment by squidfunk** [comment]
Feature released in 1.1.0
